12-add-svg-files

Toto-Tuo opened the pull request:
Chrome plugin to extract svg files
    https://chrome.google.com/webstore/detail/svg-export/naeaaedieihlkmdajjefioajbbdbdjgp?hl=en-GB
	new file:   icons/Back Button.svg
	new file:   icons/Filter Icon.svg
	new file:   icons/Powered by Onetrust.svg
	new file:   icons/Search Icon.svg
	new file:   icons/check.svg
	new file:   icons/kidney.svg
	new file:   icons/life.svg
	new file:   icons/logo.svg
	new file:   icons/member.svg
	new file:   icons/preparation.svg
	new file:   icons/svgexport-2.svg
	new file:   icons/therapy.svg

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #12-extract-and-upload-svg

Test URLs:
- Before: https://main--fresenius--hlxsites.hlx.page/
- After: https://12-add-svgs--fresenius--hlxsites.hlx.page/

Toto-Tuo committed:
12-add-svg-files
   Chrome plugin to extract svg files
    https://chrome.google.com/webstore/detail/svg-export/naeaaedieihlkmdajjefioajbbdbdjgp?hl=en-GB
	new file:   icons/Back Button.svg
	new file:   icons/Filter Icon.svg
	new file:   icons/Powered by Onetrust.svg
	new file:   icons/Search Icon.svg
	new file:   icons/check.svg
	new file:   icons/kidney.svg
	new file:   icons/life.svg
	new file:   icons/logo.svg
	new file:   icons/member.svg
	new file:   icons/preparation.svg
	new file:   icons/svgexport-2.svg
	new file:   icons/therapy.svg

helms-charity reviewed (approved):
This is great, thanks!

helms-charity reviewed (commented):
OH, actually can you rename the files so they are lower case with no spaces (use dashes)? Thank you.

Toto-Tuo committed:
rename: svg-to-lower-case
	renamed:    icons/Back Button.svg -> icons/back-button.svg
	renamed:    icons/Filter Icon.svg -> icons/filter-icon.svg
	renamed:    icons/Powered by Onetrust.svg -> icons/powered-by-onetrust.svg
	renamed:    icons/Search Icon.svg -> icons/search-icon.svg

Toto-Tuo commented:
@helms-charity 
renamed svg to lower case.
Thanks,
